pyanaconda/payload/__init__.py

```diff
@@ -76,6 +75,12 @@
 
 MAX_TREEINFO_DOWNLOAD_RETRIES = 6
 
+# Get list of repo names witch should be used as base repo
+DEFAULT_REPOS = [productName.split('-')[0].lower(),
```

Comment:
Would it make sense to have this in the main constants file so one could easily find the default repo names ? Or alternatively, I wonder if this should be even hardcoded in Anaconda VS provided as config file from some package for example, like we do with other things.

Reply:
Preferably I want to get rid of this constant at all and move this to repo metadata. This hardcoding is bad from purpose.

Moving this to configuration file won't help. The problem is the first line where we are taking base repo  based on `productName`.